internal/exec/util/file.go

```diff
@@ -77,7 +77,7 @@ func newHashedReader(reader io.ReadCloser, hasher hash.Hash) io.ReadCloser {
 // RenderFile returns a *File with a Reader that downloads, hashes, and decompresses the incoming data.
 // It returns nil if f had invalid options. Errors reading/verifying/decompressing the file will
 // present themselves when the Reader is actually read from.
-func RenderFile(l *log.Logger, c *resource.HttpClient, f types.File) *File {
+func (ut Util) RenderFile(l *log.Logger, c *resource.HttpClient, f types.File) *File {
```

Comment:
Let's go with a single-letter variable name here. Go ahead and rename the Us below to a better name (e.g. u -> user).